README.md

```diff
@@ -19,7 +19,11 @@ Enable the extension using atoum configuration file:
 
 require_once __DIR__ . DIRECTORY_SEPARATOR . 'vendor' . DIRECTORY_SEPARATOR . 'autoload.php';
 
-$runner->addExtension(new \mageekguy\atoum\ruler\extension($script));
+use mageekguy\atoum\ruler;
+
+$extension = new ruler\extension($script);
```

Comment:
the `$script` argument is required in the main process so that the extension adds the new CLI flags  (`--filter`, `--test-ext`, `--test-it`)